libuavcan_drivers/linux/include/uavcan_linux/socketcan.hpp

```diff
@@ -601,6 +601,10 @@ class SocketCanDriver : public uavcan::ICanDriver
     {
         return (iface_index >= num_ifaces_) ? nullptr : static_cast<SocketCanIface*>(ifaces_[iface_index]);
     }
+	const SocketCanIface* getIface(std::uint8_t iface_index) const override
+	{
+		return (iface_index >= num_ifaces_) ? nullptr : static_cast<const SocketCanIface*>(ifaces_[iface_index]);
+	}
```

Comment:
Just noted that this implementation is redundant, as I also mentioned in two places above.